pkg/apiclient/http1/facade.go

```diff
@@ -115,11 +126,20 @@ func (h Facade) do(ctx context.Context, in interface{}, out interface{}, method
 	log.Debugf("curl -X %s -H 'Authorization: ******' -d '%s' '%v'", method, string(data), u)
 	client := h.httpClient
 	if h.httpClient == nil {
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: h.insecureSkipVerify,
+		}
+		// Load client certificate if paths are provided
+		if h.clientCert != "" && h.clientKey != "" {
+			cert, err := tls.LoadX509KeyPair(h.clientCert, h.clientKey)
+			if err != nil {
+				return err
+			}
+			tlsConfig.Certificates = []tls.Certificate{cert}
+		}
```

Comment:
this is identical to the block added above, so a helper function probably makes sense here to DRY -- `getTLSConfig(h)` or something

Reply:
Create a helper function in `utils/tls`